DMTN-036.tex

```diff
@@ -0,0 +1,808 @@
+plane are expressed in degrees (defined through angles along great
+circles at the tangent point), so that the metric in the tangent plane
+is ortho-normal), and sidereal angles evaluated on the sky are also
+provided in degrees by the standard implementations. A notable exception
+is the LSST software stack where, by default, the angles are provided
+in radians.
+
+We now come back to the first mapping step, i.e. converting coordinates
+measured in pixel units into some intermediate coordinate system.
+The universal WCS convention here is pretty minimal: it allows for
+an affine transform, which is in general not sufficient to map the
+optical distortions of the imaging system, even after a clever choice
+of the projection. Extensions of the WCS convention have been proposed
+here, but none is universally understood. The LSST software stack
+implements the SIP addition, which consists in applying a 2-d polynomial
+transform to the CCD space coordinates, prior to entering the standard
+WCS chain (affine transform, then de-projection). In practice, the
+SIP ``twisting'' is applied by the LSST software itself (in the
+class \ClName{afw::image::TanWcs}), and the ``standard'' part
+(affine and de-projection, or the reverse transform) are sub-contracted
+to the ``libwcs'' code.
+
+One common complication of the WCS arena is that it was designed in
+the FITS framework convention, itself highly fortran-biased for array
+indexing, so that the first corner pixel of an image is indexed (1,1).
+The LSST software, and most modern environments use C-like indexing,
+i.e. images stars at (0,0), as well as coordinates in images. The
+WCS LSST software hides this detail to users, by offsetting the pixel
+space coordinates provided and obtained from the wcs-handling library.
+
+We now detail what is involved in the SIP convention: the SIP ``twisting''
+itself is encoded through 4 polynomials of 2 variables, which encode
+the direct and reverse transformations. The standard affine transform
+is expressed through a $2\times2$ matrix ($Cd$) and a reference
+point $X_{ref}$ (called \verb'CRPIX' in the fits header): 
+\[
+Y_{TP}=Cd(X_{pix}-X_{ref})
+\]
+$X_{pix}$ is a point in the CCD space, and $Y_{TP}$ is its transform
+in the tangent plane. Obviously, $X_{pix}$ and $X_{ref}$ should
+be expressed in the same frame so that the transform does not depend
+this frame choice. We write symbolically this transform as $Y_{TP}=L(X_{pix})$
+The SIP distortions are defined by a polynomial transformation in
+pixel space, that we call $P_{A}$, for the forward transformation.
+By convention, the transform from pixel space to tangent plane then
+reads: 
+\[
+Y_{TP}=L\left(X_{pix}-X_{ref}+P_{A}(X_{pix}-X_{ref})\right)
+\]
+which again does not depend on the frame choice (0-based or 1-based),
+provided $X_{pix}$ and $X_{ref}$ are expressed in the same frame.
+
+In jointcal ,the internal representation of SIP WCS's uses three straight
```

Comment:
Should "jointcal" be `\textsc{jointcal}` or something?

Reply:
I don't know. Is there a `\package{}` or equivalent defined in lsst-texmf?

Reply:
I don't think there is. We could add one. This is my MNRAS style guide persona kicking in.

Reply:
There is `\pack{}`, which I assume means `\package{}` (and which I'd suggest we rename to that...)

Reply:
I've used `\code` for all of these, which does the job.